Fixtures/Miscellaneous/TestableExe/Tests/TestableExeTests/TestableExeTests.swift

```diff
@@ -5,6 +5,13 @@ import XCTest
 import class Foundation.Bundle
 
 final class TestableExeTests: XCTestCase {
+
+  #if os(Windows)
```

Comment:
suggestion (non-blocking): Can we use `ProcessInfo.EOL` that is found in the `_InternalTestSupport` module?

See https://github.com/swiftlang/swift-package-manager/blob/main/Sources/_InternalTestSupport/Process.swift#L32-L36 for reference.

Reply:
Right.. makes sense!